File: task3/main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	const (
		xn   = 0.11
		xk   = 0.36
		step = 0.05
	)

	fmt.Println("x\t\ty")
	fmt.Println("------------------------")

	for x := xn; x <= xk+1e-9; x += step {
		y := (math.Pow(math.Sin(x), 3) + math.Pow(math.Cos(x), 3)) * math.Log(x)
		fmt.Printf("%.2f\t\t%.6f\n", x, y)
	}
}

File: task3/main.ts
const xValues = [0.2, 0.3, 0.38, 0.43, 0.57];

function calcY(x: number): number {
  return (Math.sin(x) ** 3 + Math.cos(x) ** 3) * Math.log(x);
}

console.log("x\t\ty");
console.log("------------------------");

xValues.forEach((x, i) => {
  const y = calcY(x);
  console.log(`x${i + 1} = ${x.toFixed(2)}\t${y.toFixed(6)}`);
});
